services/src/main/java/org/exoplatform/task/util/TaskUtil.java

```diff
@@ -1295,6 +1298,27 @@ public static String getResourceBundleLabel(Locale locale, String label) {
     return resourceBundleService.getResourceBundle(resourceBundleService.getSharedResourceBundleNames(), locale).getString(label);
   }
 
+  /**
+   * Gets platform language of current user. In case of any errors return null.
+   *
+   * @return the platform language
+   */
+  public static String getCurrentUserLanguage() {
+    try {
+
+      LocaleContextInfo localeCtx = LocaleContextInfoUtils.buildLocaleContextInfo(ConversationState.getCurrent().getIdentity().getUserId());
+      LocalePolicy localePolicy = ExoContainerContext.getCurrentContainer().getComponentInstanceOfType(LocalePolicy.class);
+      String lang = null;
+      if(localePolicy != null) {
+        Locale locale = localePolicy.determineLocale(localeCtx);
+        lang = locale.toString();
+      }
+      return lang;
+    } catch (Exception e) {
+      return null;
+    }
+  }
+
   /**
```

Comment:
I prefer to not have ConversationState.getCurrent in a static content. 
There are risk that it can be called by a non authenticated session.

I think you should change your method to have getUserLanguage(String userId), and the ConversationState.getCurrent is used when calling getuserLanguage

Reply:
Done!